tietokanta/src/main/resources/db/migration/V1_1999__.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE mpu_kustannustyyppi_enum AS ENUM (
+  'Arvonmuutokset',
+  'Indeksi- ja kustannustason muutokset',
+  'Muut kustannukset'
+);
+
+ALTER TABLE mpu_kustannukset ADD COLUMN kustannustyyppi mpu_kustannustyyppi_enum;
+
+-- Lisää muokkaajan tiedot, lisätty myös poistettu vaikka tätä toiminnallisuutta ei tällä hetkellä ole
+ALTER TABLE mpu_kustannukset
+ADD COLUMN luotu TIMESTAMP,
+ADD COLUMN luoja INTEGER,
```

Comment:
tähän myös references käyttäjä (id) ja ehkä not null lisäksi

Reply:
Hyvä bongaus, korjattu.